client/src/components/AddNewProfile.js

```diff
@@ -0,0 +1,125 @@
+import React, { useState } from "react";
+import { Button, Form } from "semantic-ui-react";
+
+const AddNewProfile = () => {
+  const [profileData, setProfileData] = useState({
+    firstname: "",
+    lastname: "",
+    dob: "",
+    address: "",
+    email: "",
+    phone: "",
+    gender: "",
+    groups: "",
+    support_type: ""
+  });
+
+  const handleChange = event => {
+    const updatedProfileData = {
+      ...profileData,
+      [event.target.name]: event.target.value
+    };
+
+    setProfileData(updatedProfileData);
+  };
+
+  const createProfile = () => {
+    console.log("Do something with the form values...");
+    console.log(`Username = ${profileData.firstname}`);
+    console.log(`Email = ${profileData.email}`);
+  };
+
+  return (
+    <Form>
+      <Form.Field>
+        <label>First name</label>
+        <input
```

Comment:
```suggestion
        <label for="first-name">First name</label>
        <input
          id="first-name"
```

It's good practise to link labels and inputs for these reasons:

- When you click on the label `First name` on the page, it will select the input so you can start typing
- Users who use screen readers (e.g. because they are blind) can understand that this input has the label `First name`

Let's link the labels and inputs in this whole file!

More info: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label